src/components/DevDummyLogin/index.js

```diff
@@ -162,8 +162,10 @@ const DevDummyLogin = ({resetClickTime = () => {}}) => {
       dummyLoginRbSheetRef.current.close();
     }
     const data = {appUserId, countryCode: 'ID'};
+    const password = StorageUtils.onboardingPassword.get();
+
     setDataHumenId(data, dispatch);
-    demoVerifyUser(appUserId)
+    demoVerifyUser(appUserId, password)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The password is being retrieved from storage and passed to the `demoVerifyUser` function. This could potentially be a security risk if the password is stored in plain text. Consider storing a hashed version of the password instead, or using secure storage solutions.
```diff
-    const password = StorageUtils.onboardingPassword.get();
+    const password = await SecureStore.getItemAsync('onboardingPassword');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
no problem since this is not the user's password